html/modules/custom/reliefweb_api/src/Commands/ReliefWebApiCommands.php

```diff
@@ -57,8 +57,8 @@ class ReliefWebApiCommands extends DrushCommands {
    */
   public function __construct(
     ConfigFactoryInterface $config_factory,
-    EntityFieldManager $entity_field_manager,
-    EntityTypeManager $entity_type_manager,
+    EntityFieldManagerInterface $entity_field_manager,
+    EntityTypeManagerInterface $entity_type_manager,
```

Comment:
@lazysoundsystem FYI, when injecting dependencies, the interface (if it exists, there are few exceptions in Drupal) should be used as type hint of the parameters so that the services can be overridden.

For example, the `web profiler` module replaces some services to be able to perform its profiling. Those services don't extend the Drupal service classes but implements their interfaces. In that case PHP will throw an exception if the interface is not used as type hint.